src/exercise1/final/BookClient/BookClient/AddEditBookPage.cs

```diff
@@ -96,16 +95,22 @@ async void OnDismiss(object sender, EventArgs e)
                     }
                     else
                     {
-                        Book book = await manager.Add(title, author, genre);
+                        var book = await manager.Add(title, author, genre);
                     }
 
                     await Navigation.PopModalAsync();
                 }
 
             }
+            catch(Exception ex)
```

Comment:
Not sure if it's standard style for us to omit these days, but felt like a space should be here.
```suggestion
            catch (Exception ex)
```